GVRf/Framework/src/org/gearvrf/GVRSceneObject.java

```diff
@@ -563,6 +563,26 @@ public void removeChildObject(GVRSceneObject child) {
     }
 
     /**
+     * Performs case-sensitive search
+     * 
+     * @param name
+     * @return null if nothing was found or name was null/empty
+     */
+    public GVRSceneObject[] getSceneObjectsByName(final String name) {
+        if (null == name || name.isEmpty()) {
+            return null;
```

Comment:
My goal was to filter out the name assigned by default (empty string). null could be considered an alias for this from user's perspective. Didn't want this API to be easily used as a replacement of getWholeSceneObjects.

If the user wants to use various number of spaces as names, I say let's not second guess the user.